src/components/App.test.tsx

```diff
@@ -1,11 +0,0 @@
-import * as React from 'react';
-import { shallow } from 'enzyme';
-import { App } from './App';
-
-describe('App', () => {
-  it('should match snapshot', () => {
-    const app = shallow(<App />);
-
-    expect(app).toMatchSnapshot();
-  });
-});
```

Comment:
Why removed?

Reply:
I don't think that this snapshot test added much value.
I'll add a "not exploding test" instead.